.phan/config.php

```diff
@@ -14,9 +14,11 @@
 	]
 );
 
-$cfg['suppress_issue_types'] = [
-	'PhanTypeArraySuspiciousNullable',
-	'SecurityCheck-LikelyFalsePositive',
-];
+$cfg['suppress_issue_types'] = array_merge(
```

Comment:
This shouldn't be done (unless theres a reason for this?). It was intentionally explicitly specified so that it picks up on deprecations, etc..

Reply:
I don't know why, but methods of User are considered as internal before including them.

<details>
<summary>[debug] Restarted process exited 1</summary>

```
includes/CreateWiki/CreateWikiJob.php:24 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\CreateWiki
includes/CreateWiki/CreateWikiJob.php:40 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\CreateWiki
includes/CreateWiki/CreateWikiJob.php:41 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\CreateWiki
includes/CreateWiki/CreateWikiJob.php:46 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\CreateWiki
includes/CreateWiki/SpecialCreateWiki.php:97 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\CreateWiki
includes/RequestWiki/RequestWikiAIJob.php:33 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiAIJob.php:36 PhanAccessMethodInternal Cannot access internal method \User::newSystemUser() of namespace \ defined at /var/www/html/w/includes/user/User.php:809 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiQueuePager.php:72 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiQueuePager.php:124 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiRequestViewer.php:78 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiRequestViewer.php:78 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiRequestViewer.php:112 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/RequestWikiRequestViewer.php:117 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/SpecialRequestWiki.php:34 PhanAccessMethodInternal Cannot access internal method \User::isRegistered() of namespace \ defined at /var/www/html/w/includes/user/User.php:2965 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:115 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:121 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:166 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:167 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:187 PhanAccessMethodInternal Cannot access internal method \User::getName() of namespace \ defined at /var/www/html/w/includes/user/User.php:2115 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:208 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:209 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
includes/RequestWiki/WikiRequest.php:293 PhanAccessMethodInternal Cannot access internal method \User::getId() of namespace \ defined at /var/www/html/w/includes/user/User.php:2082 from namespace \MediaWiki\Extension\CreateWiki\RequestWiki
```

</details>

Reply:
Because we are namespacing now, it isn't in the main (`\`) namespace so it isn't part of `@internal` anymore. Can you explicitly ignore the internal ones then?

Reply:
Or maybe even phan file-wide suppresions for them? Since its only two files.

Reply:
It looks like seven files to me and it's too many. I will add PhanAccessMethodInternal to config.php.

Reply:
Sounds good! Thanks!